dbt/adapters/trino/relation.py

```diff
@@ -13,7 +13,7 @@ class TrinoQuotePolicy(Policy):
 
 @dataclass(frozen=True, eq=False, repr=False)
 class TrinoRelation(BaseRelation):
-    quote_policy: TrinoQuotePolicy = TrinoQuotePolicy()
+    quote_policy: TrinoQuotePolicy = field(default_factory=lambda: TrinoQuotePolicy())
```

Comment:
Is this change related to the upgrade?

Reply:
yes, because of [this change](https://github.com/dbt-labs/dbt-core/blob/v1.4.1/core/dbt/adapters/base/relation.py#L33-L36)